documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -109,10 +109,11 @@ Let's see the options for the `index_schema_management_strategy` property:
 [options="header"]
 |===============
 |Value|Definition
-|NONE|Indexes will not be created nor deleted.
-|MERGE|Missing indexes and mappings will be created, mappings will be updated if there are no conflicts.
-|CREATE|Indexed will be deleted if existing and then created. This will delete all content from the index!
-|CREATE_DELETE|Similarly to 'CREATE' but will also delete the index at shutdown. Commonly used for tests.
+|`NONE`|Indexes will not be created nor deleted.
+|`MERGE`|Missing indexes and mappings will be created, mappings will be updated if there are no conflicts.
+|`CREATE`|**The default**: existing indexes will not be altered, missing indexes will be created.
+|`RECREATE`|Indexed will be deleted if existing and then created. This will delete all content from the index!
+|`RECREATE_DELETE`|Similarly to `RECREATE` but will also delete the index at shutdown. Commonly used for tests.
```

Comment:
Note: see org.hibernate.tool.schema.Action for a list of `hbm2ddl.auto` values and their JPA equivalent.